src/models/collection-class.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+class Collection {
```

Comment:
We can remove the collection class as we won't have that we will only have the user model and user routes which require db updates.